README.md

```diff
@@ -3,7 +3,67 @@ Contribution Guides to the CNCF Ecosystem
 
 Welcome! Are you interested in contributing to one of CNCF hosted projects? This repository should help you contains information and guidelines about contributions to CNCF projects.
 
-CNCF offers multiple ways to start contributing to the CNCF ecosystem, including either foundation-wide and project-wide opportunities.
+CNCF offers multiple ways to start contributing to the CNCF ecosystem, including either foundation-wide and project-wide opportunities. 
+
+Developers
+--------------
+
+Kubernetes: 
+[Contributor's Guide](https://github.com/kubernetes/community/tree/master/contributors/guide)
+* Example Pull Request: [Ephemeral storage monitoring via filesystem quotas](https://github.com/kubernetes/kubernetes/pull/66928)
+* Example Pull Request: [Initialize contributing cheatsheet in Bahasa Indonesia](https://github.com/kubernetes/community/pull/3697)
+
+Prometheus: [Community](https://prometheus.io/community/)
+* Example Pull Request: [GCE discovery](https://github.com/prometheus/prometheus/pull/1769)
+
+EnvoyProxy: [Community](https://www.envoyproxy.io/community)
+* Example Pull Request: [cors: add ability to enable and shadow via runtime](https://github.com/envoyproxy/envoy/pull/5265)
+
+CoreDNS: [Community](https://coredns.io/community/)
+* Example Pull Request: [plugin/metadata: add metadata plugin](https://github.com/coredns/coredns/pull/1894)
+
+Containerd: [Contributing](https://containerd.io/contributing/)
+* Example Pull Request: [Add restart manager to containerd](https://github.com/containerd/containerd/pull/2318)
+
+Fluentd: [Contributing](https://www.fluentd.org/contributing)
+* Example Pull Request: [Supported 'timezone' configuration parameter. (e.g. "+09:00" for JST)](https://github.com/fluent/fluentd/pull/430)
+
+Special interest groups within CNCF: 
+* [SIG-Storage](https://github.com/cncf/sig-storage/blob/master/storage-charter.md)
+* [SIG-Security](https://github.com/cncf/sig-security)
+
+Kubernetes [Special Interest Groups](https://github.com/kubernetes/community/blob/master/sig-list.md)
+
+
+ Designers
```

Comment:
It's a bit confusing here if we are speaking about Kubernetes contributions only or CNCF-wide? I'd suggest not to replicate the Kubernetes community content here, but add links to the original sources (to avoid the complicated fork data maintaining).